DTML.EduBot/Dialogs/TranslateDialog.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Microsoft.Bot.Builder.Dialogs;
+using Microsoft.Bot.Builder.Luis.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Web;
+
+namespace DTML.EduBot.Dialogs
+{
+    public partial class HelpDialog : QnaLuisDialog<object>
+    {
+        [LuisIntent("Translate")]
+        public async Task HandleTranslateIntent(IDialogContext context, LuisResult result)
+        {
+            await context.PostAsync("");
```

Comment:
all the `await context.PostAsync("");` statement should send back non empty messages before we can merge these changes. Can you change these?